source/libs/page-detect.ts

```diff
@@ -37,13 +37,14 @@ export const _isCommitList = [
 	'https://github.com/sindresorhus/refined-github/pull/148/commits'
 ];
 
-export const isRepoCommitList = (): boolean => /^commits\//.test(getRepoPath()!);
+export const isRepoCommitList = (): boolean => /^commits\//.test(getRepoPath()!) || (/^commits/.test(getRepoPath()!) && location.search.startsWith('?author='));
```

Comment:
I think

```suggestion
export const isRepoCommitList = (): boolean => getRepoPath()?.startsWith('commit') ?? false
```

Reply:
The issue is the this will catch pull request comits

Reply:
Aren’t they under pull/?

Reply:
AVA Failed.

Reply:
Darn it I meant `startsWith('commits')`, not `'commit'`